sdcm/utils/raft/__init__.py

```diff
@@ -331,7 +331,11 @@ def is_cluster_topology_consistent(self) -> bool:
 
 def get_raft_mode(node) -> RaftFeature | NoRaft:
     with node.parent_cluster.cql_connection_patient(node) as session:
-        return RaftFeature(node) if is_consistent_cluster_management_feature_enabled(session) else NoRaft(node)
+        raft = RaftFeature(node) if is_consistent_cluster_management_feature_enabled(session) else NoRaft(node)
+        if not raft.is_enabled:
+            LOGGER.debug("Disabling parallel nodes operations because raft is disabled")
+            node.parent_cluster.parallel_node_operations = False
+        return raft
```

Comment:
Not sure I really like the side effect in such a function

We should find a better place to use this function, and apply those changes to configure

Reply:
yes, I also wasn't happy about it. I'll try to find a better place.

Reply:
Also one day we'll need to lock down the configuration and make it immutable during test run time.

We can apply the logic just before we gonna use the flag configuration

Reply:
Now I see it's not configuration...  but still confusing to some degree, cause there is such configuration

Reply:
I removed it. Actually we need to disable `parallel_node_operations` when we force gossip - when this config param will be introduced it should be taken into account. Otherwise, version&config base should be enough for now.